lib/cryptoexchange/exchanges/coinex/services/market.rb

```diff
@@ -43,7 +43,7 @@ def adapt(output, market_pair)
           ticker.last      = NumericHelper.to_d(output['last'])
           ticker.high      = NumericHelper.to_d(output['high'])
           ticker.low       = NumericHelper.to_d(output['low'])
-          ticker.volume    = NumericHelper.to_d(output['volume'])
+          ticker.volume    = NumericHelper.to_d(output['vol'])
```

Comment:
coinex changed their api. Please refer to api url below:-

https://api.coinex.com/v1/market/ticker/all